File: day01/dart/day01.dart
import 'dart:io';

int calcYear(int year) {
  return year * 365;
}

main() {
  // int year = 300;
  // print("Qual o ano: ");
  var year = stdin.readLineSync();
  print("Anos 300 em dias: ${calcYear(int.parse(year))}");
}

// dart run day01.dart

File: day01/go/day01.go
package main

import "fmt"

func calcYear(year int) int {
	year = year * 365
	return year
}

func main() {
	year := 0
	fmt.Println("Digite o ano")
	fmt.Scanln(&year)
	fmt.Println("Ano: ", year, " Dias:", calcYear(year))

}

// go run day01.go
